init-script

```diff
@@ -27,6 +27,7 @@ echo "Running Mer Boat Loader"
 BOOTLOGO=%BOOTLOGO%
 ALWAYSDEBUG=%ALWAYSDEBUG%
 DATA_PARTITION=%DATA_PART%
+SYSTEM_PARTITION=$(findfs LABEL="system")
```

Comment:
Probably, not working on all devices, only on my device it's automatically labeled

Reply:
Also on mines, I think this is always labelled